src/components/Movies/Movies.jsx

```diff
@@ -0,0 +1,70 @@
+
+import React, { useState, useEffect, useCallback } from "react";
+
+import { useSelector, useDispatch } from 'react-redux';
+
+import Movie from "../Movie/Movie";
+import '../../App.css';
+
+import { fetchMovies } from '../store/thunk/LoadingMovieList';
+
+function Movies(props) {
+
+  const [value, setValue] = useState(1);
+
+  const { items, setItems, findFilms } = props;
+
+  const dispatch = useDispatch();
+
+  const status = useSelector(state => state.AddMovies.status);
+
+  const error = useSelector(state => state.AddMovies.error);
+
+  const item = useSelector(state => state.AddMovies.films)
```

Comment:
write useSelectors properly
and we don't name field with a capital letter AddMovies - addMovies